### ISSUE TITLE
E0091 needs to be updated to new format

### ISSUE THREAD

**Issue by jonathandturner** [rust-lang/rust#35228]
From: src/test/compile-fail/E0091.rs

Error E0091 needs a span_label, updating it from:

```
error[E0091]: type parameter `T` is unused
  --> src/test/compile-fail/E0091.rs:11:10
   |
11 | type Foo<T> = u32; //~ ERROR E0091
   |          ^

error[E0091]: type parameter `B` is unused
  --> src/test/compile-fail/E0091.rs:12:14
   |
12 | type Foo2<A, B> = Box<A>; //~ ERROR E0091
   |              ^
```

To:

```
error[E0091]: type parameter `T` is unused
  --> src/test/compile-fail/E0091.rs:11:10
   |
11 | type Foo<T> = u32; //~ ERROR E0091
   |          ^ unused

error[E0091]: type parameter `B` is unused
  --> src/test/compile-fail/E0091.rs:12:14
   |
12 | type Foo2<A, B> = Box<A>; //~ ERROR E0091
   |              ^ unused
```

**Comment by Vassah** [comment]
I'll be tackling this one over the next few days.

**Comment by Vassah** [comment]
Addressed in [#35528](https://github.com/rust-lang/rust/pull/35528) pending merge.
